tests/e2e/baseTest.ts

```diff
@@ -291,3 +302,50 @@ async function globalBeforeEach(page: Page, electronApp: ElectronApplication): P
   const notification = new Notification(page, infoNotifications.first());
   await notification.dismiss();
 }
+
+async function globalAfterEach(
+  testTempDir: string,
+  electronApp: ElectronApplication,
+  page: Page,
+  testInfo: TestInfo,
+): Promise<void> {
+  const notificationArea = new NotificationArea(page);
+
+  // store the extension logs
+  const extensionLogPath = path.join(testTempDir, "vscode-confluent.log");
```

Comment:
I started with downloading the support zip directly, but we have to deal with enough zip downloads-and-unzips when debugging tests that it felt like overkill here.